packages/expo-google-sign-in/src/index.js

```diff
@@ -0,0 +1,337 @@
+// @flow
+import { NativeModulesProxy } from 'expo-core';
+
+const { ExpoGoogleSignIn } = NativeModulesProxy;
+
+export const { ERRORS, SCOPES, TYPES } = ExpoGoogleSignIn;
+
+export type GoogleSignInType = TYPES.DEFAULT | TYPES.GAMES;
+
+export type GoogleSignInOptions = {
+  scopes: ?Array<string>,
+  webClientId: ?string,
+  hostedDomain: ?string,
+  accountName: ?string,
+
+  // Android
+  signInType: ?SignInType,
+  isOfflineEnabled: ?boolean,
+  isPromptEnabled: ?boolean,
+  // iOS
+  clientId: ?string,
+  language: ?string,
+  openIdRealm: ?string,
+};
+
+
+export class AuthData {
+  constructor() {
+    this.equals = this.equals.bind(this);
+    this.toJSON = this.toJSON.bind(this);
+  }
+
+  equals(other: ?any): boolean {
+    if (!other || !(other instanceof AuthData)) return false;
+    return true;
+  }
+
+  toJSON() {
+    return {};
+  }
+}
+export class Identity extends AuthData {
```

Comment:
Could you, please, split this one big file into multiple files to make it a bit more readable? 🙂 Also, there are some formatting issues. 💅